pubspec.yaml

```diff
@@ -12,3 +12,6 @@ environment:
 dev_dependencies:
   dart_flutter_team_lints: ^2.0.0
   test: ^1.16.0
+
+topics:
```

Comment:
As a nit, I normally try and keep a rough order to pubspec fields (critical meta-data like name, version, other entries, and then regular deps, dev deps, ...). So, I generally put `topics` after the main metadata, but before the deps sections.

eg. https://github.com/dart-lang/lints/blob/main/pubspec.yaml